internal/ioutil/print.go

```diff
@@ -52,3 +53,28 @@ func PrintCertificateMap(w io.Writer, v []config.CertificateReference) error {
 	}
 	return tw.Flush()
 }
+
+func PrintVerificationResults(w io.Writer, v []*verification.SignatureVerificationOutcome, resultErr error, digest string) error {
+	tw := newTabWriter(w)
+
+	if resultErr == nil {
+		fmt.Fprintf(tw, "Signature verification succeeded for %s\n", digest)
+		// TODO: print out failed validations as warnings.
```

Comment:
Can you please also add issue with TODO: https://github.com/notaryproject/notation/issues/304
Would allow easier tracking.